examples/KitchenSink/.gitignore

```diff
@@ -1,4 +1,4 @@
 !/logs/.gitkeep
 /logs/*
-!/public/static/tmpdir/.gitkeep
```

Comment:
this rule behave unexpectedly. `tmpdir/gitkeep` is ignored.
so this file open process of KitchenSink is failure, and resource not found.

https://github.com/line/line-bot-sdk-php/blob/master/examples/KitchenSink/src/LINEBot/KitchenSink/EventHandler/MessageHandler/ImageMessageHandler.php#L77